src/vos/tests/vos_dfs_sample.yaml

```diff
@@ -1,34 +1,111 @@
 ---
 # Sample conflig file DFS files and directories
 num_pools: 1000
-time_key: &time
-  count: 3
-  size: 5
+
+dfs_magic: &magic
+  type: integer
```

Comment:
According to the README, the akey is "DFS_MAGIC" so not an integer key.

@mchaarawi Perhaps we should make it an integer enumeration as it would save some space and make lookups a little more efficient (integer compare vs string compare).   But as it stands, this is not correct.   It should be 9 bytes.   Similar comment for the akeys below.   Same for DFS_SB_METADATA

Reply:
well this is just the Superblock object in the container.
The keys are never enumerated and only written on create, and fetched/verified on mount.